kpops/cli/main.py

```diff
@@ -345,5 +353,13 @@ def clean(
         component.clean(dry_run)
 
 
+@app.callback()
+def main(
+        version: bool = typer.Option(None, "--version", callback=version_callback, is_eager=True),
+):
+    log.debug(f"Version option called: {version}")
```

Comment:
I don't think this is necessary. @disrupted WDYT?

Reply:
yes, the debug logging is unnecessary

Reply:
removed the unnecessary logging statement